spiketoolkit/sorters/ironclust/ironclust.py

```diff
@@ -1,7 +1,163 @@
+from pathlib import Path
+import os
+
+from ..tools import  _call_command_split
+from spiketoolkit.sorters.basesorter import BaseSorter
+import spikeextractors as se
+
+try:
+    from mountainlab_pytools import mdaio
+    HAVE_IRONCLUST = True
+except ModuleNotFoundError:
+    HAVE_IRONCLUST = False
+
+
+class IronclustSorter(BaseSorter):
+    """
+    """
+    
+    sorter_name = 'ironclust'
+    installed = HAVE_IRONCLUST
+    # SortingExtractor_Class = se.NumpySortingExtractor
+    SortingExtractor_Class = None # custum get_result
+    
+    _default_params = {
+        'prm_template_name': None,  # Name of the template file
+        'detect_sign': -1,  # Polarity of the spikes, -1, 0, or 1
+        'adjacency_radius': -1,  # Channel neighborhood adjacency radius corresponding to geom file
+        'detect_threshold': 5,  # Threshold for detection
+        'merge_thresh': .98,  # Cluster merging threhold 0..1
+        'freq_min': 300,  # Lower frequency limit for band-pass filter
+        'freq_max': 6000,  # Upper frequency limit for band-pass filter
+        'pc_per_chan': 3,  # Number of pc per channel
+        'parallel': True
+    }
+
+    installation_mesg = """
+       >>> pip install mountainlab_pytools
```

Comment:
can you add: `kbucket` to the pip install? That is also needed for ironclust

Reply:
with pip install kbucket ?